src/value.rs

```diff
@@ -132,3 +141,33 @@ pub fn type_from_str(name: &str) -> Option<RbxValueType> {
         _ => None,
     }
 }
+
+struct Color3Value {
+    value: [f32; 3],
+}
+
+impl Color3Value {
+    pub fn new(value: [f32; 3]) -> Self {
+        Self { value }
+    }
+
+    fn meta_to_string<'lua>(&self, context: Context<'lua>) -> rlua::Result<rlua::Value<'lua>> {
+        format!("{}, {}, {}", self.value[0], self.value[1], self.value[2]).to_lua(context)
+    }
```

Comment:
Let's port this code to a `Display` impl on the type! Then, when we implement the `__tostring` metamethod below, we can do it as `Ok(this.to_string())`.